Gemfile

```diff
@@ -5,23 +5,24 @@ ruby '2.6.3'
 gem 'rails', '~> 6.0.0'
 
 gem 'active_storage_base64', '~> 1.0.0'
-gem 'activeadmin', '~> 2.2'
+gem 'activeadmin', '~> 2.4'
 gem 'aws-sdk-s3', '~> 1', require: false
-gem 'bootsnap', '~> 1.3.0'
-gem 'delayed_job_active_record', '~> 4.1.3'
+gem 'bootsnap', '~> 1.4', '>= 1.4.5'
+gem 'delayed_job_active_record', '~> 4.1', '>= 4.1.4'
 gem 'devise', '~> 4.7.1'
-gem 'devise_token_auth', '~> 1.1.2'
+gem 'devise_token_auth', '~> 1.1', '>= 1.1.3'
 gem 'draper', '~> 3.1'
 gem 'figaro', '~> 1.1.1'
-gem 'haml-rails', '~> 1.0.0'
+gem 'haml-rails', '~> 2.0', '>= 2.0.1'
 gem 'jbuilder', '~> 2.9.1'
-gem 'oj', '~> 3.7', '>= 3.7.12'
-gem 'pg', '~> 0.18.2'
-gem 'pry-rails', '~> 0.3.6'
-gem 'puma', '~> 3.0'
-gem 'rack-cors', '~> 0.4.0'
+gem 'oj', '~> 3.9', '>= 3.9.2'
+gem 'pg', '~> 1.1', '>= 1.1.4'
+gem 'pry-rails', '~> 0.3.9'
+gem 'puma', '~> 4.3'
+gem 'rack-cors', '~> 1.0', '>= 1.0.4'
```

Comment:
![image](https://user-images.githubusercontent.com/6373536/68953710-4101fd80-07a1-11ea-8e27-f6d6cf511800.png)

For some reason version 1.0.4 was yanked, we'll need to upgrade it

Reply:
Yep, done

Reply:
https://nvd.nist.gov/vuln/detail/CVE-2019-18978